census_view.hpp

```diff
@@ -38,61 +38,51 @@
 #include <vector>
 
 class CensusDocument;
-class CensusViewDataViewModel;
-
-class WXDLLIMPEXP_FWD_ADV wxDataViewEvent;
-class WXDLLIMPEXP_FWD_ADV wxDataViewCtrl;
 
-class CensusView final
+class CensusView
     :public ViewEx
 {
-    friend class CensusDocument;
-    friend class CensusViewDataViewModel;
-
   public:
     CensusView();
 
-  private:
+  protected:
```

Comment:
It seems strange to make deleted copy ctor/assignment operator protected. They probably should remain private.

Which methods do we actually need to be protected?